Firmware/Marlin_main.cpp

```diff
@@ -4015,7 +4015,7 @@ eeprom_update_word((uint16_t*)EEPROM_NOZZLE_DIAMETER_uM,0xFFFF);
                strchr_pointer++;                  // skip 1st char (~ 's')
                strchr_pointer++;                  // skip 2nd char (~ 'e')
                nDiameter=(uint16_t)(code_value()*1000.0+0.5); // [,um]
-               eeprom_update_byte((uint8_t*)EEPROM_NOZZLE_DIAMETER,(uint8_t)ClNozzleDiameter::_Diameter_Undef); // for correct synchronization after farm-mode exiting
+               eeprom_update_byte_notify((uint8_t*)EEPROM_NOZZLE_DIAMETER,(uint8_t)ClNozzleDiameter::_Diameter_Undef); // for correct synchronization after farm-mode exiting
```

Comment:
This can't be changed during print, so maybe leave it and let the host to read it
- after connect
- before starting print
- ???

Reply:
It makes sense to report changes even when the user operates the LCD menu, hosts like Octoprint may update their UI in sync with what is happening directly on the printer. Moreover, we are not that limited in terms of CPU cycles when not printing (i.e. we may post more notifications).